composer.json

```diff
@@ -16,7 +16,7 @@
 	},
 
 	"require": {
-		"php": "^5.3|^7.0"
+		"php": "^5.4|^7.0"
```

Comment:
I'll take your advice into consideration, but won't act on it at the moment, because people use EOL versions of PHP anyway.

Please revert this change.